src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -3136,7 +3149,31 @@ NetworkOPsImp::transJson(
         }
     }
 
-    return jvObj;
+    MultiApiJson multiObj({jvObj, jvObj});
+    // Minimum supported API version must match index 0 in MultiApiJson
+    static_assert(apiVersionSelector(RPC::apiMinimumSupportedVersion)() == 0);
+    // Beta API version must match last index in MultiApiJson
+    static_assert(
+        apiVersionSelector(RPC::apiBetaVersion)() + 1  //
+        == MultiApiJson::size);
+    for (unsigned apiVersion = RPC::apiMinimumSupportedVersion,
```

Comment:
For the intentions of looping over the apiVersions from `minimumSupported` to `beta`. I actually made a helper function long time back - may/ may not be useful :) 
https://github.com/XRPLF/rippled/pull/4611

Reply:
That function is cool, but as written, it's only for tests.

Reply:
oh shoot, my bad. I did not notice this was not for tests 😆

Reply:
No worries